_includes/data/claims_data_avail.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  <p>Since the Beneficiary Claims Data API (BCDA) shares fully adjudicated Medicare claims data (Part A, B, and D), claims data availability relies on how quickly a claim has been submitted, processed, and approved. Per Section 6404 of the Affordable Care Act, the maximum period for submission of all Medicare Fee-for-Service claims has been reduced to no more than 12 months (1 calendar year) after the date services were furnished. CMS typically receives claims 3-4 weeks after care has been provided. Once received by CMS, it is possible for claims to undergo more than one round of processing to make adjustments, edits, and cancellations. Data will only be available via the API once a claim has been approved. For more details on claims submission and approval timeframes, please review this <a href="https://www2.ccwdata.org/documents/10280/19002256/medicare-claims-maturity.pdf" target="_blank" rel="noopener" class="in-text__link">white paper</a> which outlines the lifecycle of a Medicare claim, as well as timeframes for submission and approval (pages 11, 12, and 22 have details on month-over-month statistics). 
+  <p>Since the Beneficiary Claims Data API (BCDA) shares fully adjudicated Medicare claims data (Part A, B, and D), claims data availability relies on how quickly a claim has been submitted, processed, and approved. Per Section 6404 of the Affordable Care Act, the <a href="https://www.hhs.gov/guidance/document/systems-changes-necessary-implement-patient-protection-and-affordable-care-act-ppaca-1#:~:text=and%20transmitted%20securely.-,Systems%20Changes%20Necessary%20to%20Implement%20the%20Patient%20Protection%20and%20Affordable,Not%20More%20Than%2012%20Months" target="_blank" rel="noopener" class="in-text__link">maximum period for submission of all Medicare Fee-for-Service claims</a> has been reduced to no more than 12 months (1 calendar year) after the date services were furnished. CMS typically receives claims 3-4 weeks after care has been provided. Once received by CMS, it is possible for claims to undergo more than one round of processing to make adjustments, edits, and cancellations. Data will only be available via the API once a claim has been approved. For more details on claims submission and approval timeframes, please review this <a href="https://www2.ccwdata.org/documents/10280/19002256/medicare-claims-maturity.pdf" target="_blank" rel="noopener" class="in-text__link">white paper</a> which outlines the lifecycle of a Medicare claim, as well as timeframes for submission and approval (pages 11, 12, and 22 have details on month-over-month statistics). 
```

Comment:
I couldnt find a specific link to section 6404 but did find this link which seems to be the point of the sentence generally: https://www.hhs.gov/guidance/document/systems-changes-necessary-implement-patient-protection-and-affordable-care-act-ppaca-1#:~:text=and%20transmitted%20securely.-,Systems%20Changes%20Necessary%20to%20Implement%20the%20Patient%20Protection%20and%20Affordable,Not%20More%20Than%2012%20Months